### PR TITLE
Removed some cases of variable set but not used

### LINKED ISSUE #836: Build warnings when running cmake.
I just did a build of the master branch d1d9839b83800ac04eb4a0bb50d64e2c3efed63c with cmake  following the instructions in the "Compilation" section of README.md, and got the following compiler warnings. (I edited out the cmake progress messages.) This is with gcc 11.2.0 in Linux.

```
/home/warren/repos/git/forks/HiGHS/src/lp_data/Highs.cpp:2842:12: warning: variable 'postsolve_iteration_count' set but not used [-Wunused-but-set-variable]
  HighsInt postsolve_iteration_count = -1;
           ^
1 warning generated.


/home/warren/repos/git/forks/HiGHS/src/mip/HighsDomain.cpp:2403:16: warning: variable 'propnnz' set but not used [-Wunused-but-set-variable]
      HighsInt propnnz = 0;
               ^
/home/warren/repos/git/forks/HiGHS/src/mip/HighsDomain.cpp:2496:18: warning: variable 'propnnz' set but not used [-Wunused-but-set-variable]
        HighsInt propnnz = 0;
                 ^
2 warnings generated.


/home/warren/repos/git/forks/HiGHS/src/mip/HighsCliqueTable.cpp:1667:18: warning: variable 'numDel' set but not used [-Wunused-but-set-variable]
        HighsInt numDel = 0;
                 ^
1 warning generated.


/home/warren/repos/git/forks/HiGHS/src/presolve/HighsLpPropagator.cpp:485:14: warning: variable 'propnnz' set but not used [-Wunused-but-set-variable]
    HighsInt propnnz = 0;
             ^
1 warning generated.


/home/warren/repos/git/forks/HiGHS/src/presolve/Presolve.cpp:188:12: warning: variable 'total_rows' set but not used [-Wunused-but-set-variable]
  HighsInt total_rows = 0;
           ^
/home/warren/repos/git/forks/HiGHS/src/presolve/Presolve.cpp:1027:12: warning: variable 'cntEl' set but not used [-Wunused-but-set-variable]
  HighsInt cntEl = 0;
           ^
2 warnings generated.


In file included from /home/warren/repos/git/forks/HiGHS/src/qpsolver/quass.cpp:13:
In file included from /home/warren/repos/git/forks/HiGHS/src/qpsolver/feasibility.hpp:7:
/home/warren/repos/git/forks/HiGHS/src/qpsolver/feasibility_quass.hpp:31:13: warning: empty parentheses interpreted as a function declaration [-Wvexing-parse]
 Basis basis();
            ^~
/home/warren/repos/git/forks/HiGHS/src/qpsolver/feasibility_quass.hpp:31:13: note: replace parentheses with an initializer to declare a variable
 Basis basis();
            ^~
            {}
1 warning generated.



/home/warren/repos/git/forks/HiGHS/src/util/HighsUtils.cpp:538:12: warning: variable 'sumK' set but not used [-Wunused-but-set-variable]
  HighsInt sumK = 0;
           ^
/home/warren/repos/git/forks/HiGHS/src/util/HighsUtils.cpp:541:12: warning: variable 'sumPct' set but not used [-Wunused-but-set-variable]
  HighsInt sumPct = 0;
           ^
/home/warren/repos/git/forks/HiGHS/src/util/HighsUtils.cpp:722:10: warning: variable 'sum_percentage' set but not used [-Wunused-but-set-variable]
  double sum_percentage = 0;
         ^
3 warnings generated.
```

### PR DESCRIPTION
**Pull request by jajhall**
Once @feldmeier has considered one of the points raised, this will close #836

### CODE REVIEW

File: src/lp_data/Highs.cpp [hunk 1]
```diff
@@ -2874,14 +2873,12 @@ HighsStatus Highs::callRunPostsolve(const HighsSolution& solution,
     // Set up the iteration count and timing records so that
     // adding the corresponding values after callSolveLp gives
     // difference
-    postsolve_iteration_count = -info_.simplex_iteration_count;
```
Comment (WarrenWeckesser) [inline src/lp_data/Highs.cpp:2877]: Shouldn't the preceding comment ("Set up the iteration count...") also be updated?

File: src/lp_data/Highs.cpp [hunk 2]
```diff
@@ -2874,14 +2873,12 @@ HighsStatus Highs::callRunPostsolve(const HighsSolution& solution,
...
     timer_.start(timer_.solve_clock);
     call_status = callSolveLp(
         incumbent_lp,
         "Solving the original LP from the solution after postsolve");
     timer_.stop(timer_.solve_clock);
     // Determine the iteration count and timing records
-    postsolve_iteration_count += info_.simplex_iteration_count;
```
Comment (WarrenWeckesser) [inline src/lp_data/Highs.cpp:2884]: Same comment here--it looks like removing this line makes part of the preceding comment obsolete.

File: src/lp_data/Highs.cpp (hunk 1 above)
Comment (jajhall) [reply src/lp_data/Highs.cpp:2877]: Of course

File: src/lp_data/Highs.cpp (hunk 2 above)
Comment (jajhall) [reply src/lp_data/Highs.cpp:2884]: Ditto

**Pull request merged**